pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -120,163 +120,21 @@ func ExecuteXlInfoCmd() map[string]string {
 	return dict
 }
 
-// Shadow copy of suscription to determine info for deletes. Key is UUID
-var domainStatus map[string]types.DomainStatus
+// Helper function
+func getAppInterfaceList(aiStatus types.AppInstanceStatus) []string {
```

Comment:
This looks like it should be a method of struct AppInstanceStatus ?? 
     aiStatus.getAppInterfaceList()

Reply:
Fixed